MoveImage2/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Text;
+using Form;
+
+namespace Form
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.Write("Input console color: ");
+
+            string foregrcolor = Console.ReadLine();
+            Color consColor = new Color(foregrcolor);
+            consColor.GetConsoleColor();
+
+            Console.Write("Choose your image type(1-Rectangle, 2-Triangle): ");
+            int type = Convert.ToInt32(Console.ReadLine());
+            if (type == 1)
+            {
+                RectangleBuilder rectangleBuilder = new RectangleBuilder();
+                rectangleBuilder.CreateRectangle(type);
```

Comment:
Same comment as above. Consider using interface for builders. Also there is no need for form type checking in `moveImage.Print` since you are doing checking here.